content/ecosystem/_index.html

```diff
@@ -0,0 +1,461 @@
+---
+title: RAPIDS.ai | Ecosystem
+description: RAPIDS libraries, frameworks, extensions and more
+---
+
+<!-- Hero Shortcode -->
+{{< hero hero-text="Ecosystem" button-link="#software" button-text="Go to Software">}}
+
+  <!-- Intro -->
+  <section class="section mb-6">
+
+    <div class="container">
+      <h1 class="has-text-centered mb-5"> An ecosystem of hardware, <br> software, and developers</h1>
+      <div class="columns">
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-microchip"></i> Hardware </h2>
+          <p> RAPIDS runs on hardware powered by CUDA, the fundamental set of tools for GPU computing built by
+            NVIDIA. RAPIDS works on PCs, workstations, data centers, and the cloud. Learn more about the different types
+            of hardware and find the best fit for your use case on NVIDA's product page for <a href="https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/"> Data Science Hardware <i class="fa-solid fa-arrow-up-right"></i></a> </p>
+
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-code"></i> Software </h2>
+          <p> Set on a foundation of powerful hardware and parallelized by CUDA, the RAPIDS ecosystem is based on core
+            projects, those extending its capabilities, and those built on top of RAPIDS.</p>
+            <a href="#featured-software">Jump to Section <i class="fa-solid fa-arrow-up-right"></i></a>
+
+          <br>
+          <img class="image img-center mt-5" src="/images/rapid-layers.png">
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-laptop-code"></i> Developers </h2>
+          <p> We hope you will consider integrating RAPIDS into your solution or workflow. RAPIDS is for developers ranging from competent python users to C++ experts. Build something great with RAPIDS.</p> <a href="#developers">Jump to Section <i class="fa-solid fa-arrow-up-right"></i></a>
+
+        </div>
+      </div>
+    </div>
+
+  </section>
+
+  <!-- Software RAPIDS Libs  -->
+  <div id="featured-software"></div>
+  <div class="top-slant-container">
+    <div class="top-slant-down has-background-rapids-purple-primary"></div>
+  </div>
+
+  <section class="section has-background-rapids-purple-primary">
+    <div class="container">
+
+      <h1 class="has-text-white">Featured RAPIDS</h1>
```

Comment:
Feels like this is missing something--should this be RAPIDS Libraries?

```suggestion
      <h1 class="has-text-white">RAPIDS Libraries</h1>
```

Reply:
Its technically not all of them - Featured RAPIDS Libraries